index.bs

```diff
@@ -6152,14 +6132,21 @@ to grow organically as needed.
  1. Perform ! [$SetUpReadableStreamDefaultController$](|stream|, |controller|, |startAlgorithm|,
     |pullAlgorithmWrapper|, |cancelAlgorithmWrapper|, |highWaterMark|, |sizeAlgorithm|).
 
- <p class="note">This algorithm is mainly intended for use in the [=constructor operations=] of any
- {{ReadableStream}} subclasses.
+ <div class="example" id="example-set-up-rs">
+  Creating a {{ReadableStream}} from other specifications is thus a two-step process, like so:
+
+  1. Let |readableStream| be a [=new=] {{ReadableStream}}.
+  1. [=ReadableStream/Set up=] |readableStream| given….
```

Comment:
This needs to say something about how to use _readableStream_ in the creation of the algorithms.